src/pk/ecc/ecc_verify_hash.c

```diff
@@ -76,11 +76,11 @@ int ecc_verify_hash_ex(const unsigned char *sig,  unsigned long siglen,
    }
    else if (sigformat == LTC_ECCSIG_RFC7518) {
       /* RFC7518 format - raw (r,s) */
-      i = mp_unsigned_bin_size(key->dp.order);
-      if (siglen != (2*i)) {
+      if (siglen % 2) {
```

Comment:
probably revert again to the original version `if ((siglen % 2) == 1)`? I'm not entirely sure why, but I don't like the implicit to-boolean conversion of C anymore...

Reply:
Done. I believe clang-tidy will like `if ((siglen % 2) == 1)` more as well.